src/example/browser/simple-browser-support.coffee

```diff
@@ -1,6 +1,6 @@
 window.cql = require '../../cql'
 
-window.executeSimpleELM = (elm, patientSource, valueSets, libraryName, version, parameters = {}) ->
+window.executeSimpleELM = (elm, patientSource, valueSets, libraryName, version, executionDateTime, parameters = {}) ->
```

Comment:
So... in this same directory, there is an example HTML file that invokes `executeSimpleELM`.  It seems it's probably been broken for a while (not your fault), but can you update it to invoke `executeSimpleELM` with the right function signature?  It would be good to have an example of how the `executionDateTime` should be passed in.

https://github.com/cqframework/cql-execution/blob/ea6806eccf059722ded252d1a66a4ef535280f3d/src/example/browser/cql4browsers.html#L13

Reply:
I am thinking it might be better for Bonnie to create a new entry point into the engine (somewhere not under `example`) And then we change this file back to what it was before. Thoughts? I would say this would also be a different PR.

Reply:
I think it probably would be good for Bonnie dependencies not to be in an _example_ directory.  That said, I think it would also be good for the example to show how to do this. ;-)

But if it's trickier than it looks, just go ahead and leave it like this for this PR (since I think it's been broken for a while).

Reply:
Completely agree! I'll take a crack at it and see if I can get it working without too much effort.